src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Main {
```

Comment:
The `Main` class should not be abstract. Remove the `abstract` keyword to allow the `main` method to be executed as the entry point of the application.